src/Queue/DoctrineQueue.php

```diff
@@ -417,7 +407,7 @@ protected function parseOptionsToDateTime($options)
                     try {
                         // first try ISO 8601 duration specification
                         $delay = new DateInterval($options['delay']);
-                    } catch (\Exception $e) {
+                    } catch (Exception $e) {
```

Comment:
Personally I would rather keep the `\`, so I can see that this one is from the standard library. But probably boils down to taste.